src/parameters.toml

```diff
@@ -935,155 +1008,60 @@ value = 0.115231
 type = "float"
 description = "q coefficinet for raindrop terminal velocity parameterization [-]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
 
-[Chen2022_table_B1_a1_coeff]
-value = 0.044612
+[Chen2022_table_B1_ai]
+value = [0.044612, -0.263166, 4.7178]
 type = "float"
-description = "a1 coefficinet for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
-
-[Chen2022_table_B1_a2_coeff]
-value = -0.263166
-type = "float"
-description = "a2 coefficinet for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
-
-[Chen2022_table_B1_a3_coeff]
-value = 4.7178
-type = "float"
-description = "a3 coefficinet for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
+description = "ai coefficinets for raindrop terminal velocity parameterization [mm^-bi]. See Table B1 in Chen et al 2022 DOI: 10.1016/j.atmosres.2022.106171"
```

Comment:
There is a typo `coefficinets` -> `coefficients` in many descriptions. It would be good to fix those quickly as well!

Reply:
As long as I add less typos than I remove, things should continue to improve ;) Thank you for caching that. Should be fixed now.